src/fiskaltrust.ifPOS/v1/de/IDESSCD.cs

```diff
@@ -11,38 +11,92 @@ public interface IDESSCD
     {
         [OperationContract]
 #if WCF
-        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/starttransactionexportdata", Method = "POST")]
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/starttransaction", Method = "POST")]
 #endif
-        Task<StartTransactionResponse> StartTransactionExportDataAsync(StartTransactionRequest request);
+        Task<StartTransactionResponse> StartTransactionAsync(StartTransactionRequest request);
 
         [OperationContract]
 #if WCF
-        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/updatetransactionexportdata")]
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/updatetransaction", Method = "POST")]
 #endif
-        Task<UpdateTransactionResponse> UpdateTransactionExportDataAsync(UpdateTransactionRequest request);
+        Task<UpdateTransactionResponse> UpdateTransactionAsync(UpdateTransactionRequest request);
 
         [OperationContract]
 #if WCF
-        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/finishtransactionexportdata")]
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/finishtransaction", Method = "POST")]
 #endif
-        Task<FinishTransactionResponse> FinishTransactionExportDataAsync(FinishTransactionRequest request);
+        Task<FinishTransactionResponse> FinishTransactionAsync(FinishTransactionRequest request);
+
 
```

Comment:
It would be great if we could get rid of all the double newlines and replace them with normal ones :+1: